pkg/catalog/manager/manager.go

```diff
@@ -242,7 +242,13 @@ func (m *Manager) ValidateKubeVersion(template *v3.CatalogTemplateVersion, clust
 	return nil
 }
 
-func (m *Manager) ValidateRancherVersion(template *v3.CatalogTemplateVersion) error {
+func (m *Manager) ValidateRancherVersion(template *v3.CatalogTemplateVersion, currentAppVersion string) error {
+	if currentAppVersion != "" && currentAppVersion == template.Spec.Version {
```

Comment:
nit: I think `currentAppVersion` should be a `*string`, where `nil` indicates that a current app version does not exist.

Reply:
We could use a pointer but I'm not sure if it is wise to use pointers where they are no necessary as it invites another point of failure. An empty string indicating that it isn't being used is not much different than using nil.

Reply:
hmm agreed. resolving, no need to change